satellite/payments/paymentsconfig/config.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package paymentsconfig
+
+import "storj.io/storj/satellite/payments/stripecoinpayments"
+
+// Config defines global payments config.
+type Config struct {
+	Provider           string `help:"payments provider to use" default:""`
+	StripeCoinpayments stripecoinpayments.Config
```

Comment:
I'm wondering whether the correct capitalization is `CoinPayments` or `Coinpayments`?

Reply:
so `coinPayments` if it local variable or so

Reply:
yeah